run_ci.sh

```diff
@@ -14,9 +14,13 @@ function getlogs(){
     sudo podman logs coreos-downloader > $LOGDIR/coreos-downloader.log
     sudo podman logs ipa-downloader > $LOGDIR/ipa-downloader.log
 
-    # And the VM jornals
+    # And the VM journals and staticpod container logs
     for HOST in $(sudo virsh net-dhcp-leases baremetal | grep -o '192.168.111.[0-9]\+') ; do
         sshpass -p notworking $SSH core@$HOST sudo journalctl > $LOGDIR/$HOST-system.journal || true
+        sshpass -p notworking $SSH core@$HOST sudo journalctl -u ironic.service > $LOGDIR/$HOST-ironic.journal || true
+	for c in $(sudo podman ps -a | grep -e ironic -e downloader -e httpd -e dnsmasq -e mariadb | awk '{print $NF}'); do
```

Comment:
is "podman ps -a ...." being run on the wrong host here, should be on $HOST

Reply:
Ah yes, good catch - thanks!